code/modules/mob/living/carbon/monkey/roach.dm

```diff
@@ -0,0 +1,21 @@
+/mob/living/carbon/monkey/roach
+	name = "roach"
+	voice_name = "roach"
+	speak_emote = list("hisses")
+	icon_state = "bigroach"
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/roach/big
+	species_type = /mob/living/carbon/monkey/rock
```

Comment:
???

Reply:
ah, thats a good catch, i based their code on the golem monkies